message-index/messages/GHC-64725/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: User-defined type error
+summary: A constraint or type family reduced to a custom type error.
```

Comment:
```suggestion
summary: An invalid constraint or type family reduces to a custom type error.
```

I'm not sure whether this is a slightly better reading? I think the summary should be "this is why the error occurred" rather than "this is *what* the error is"